db/migrate/20170127000826_add_outcomes.rb

```diff
@@ -1,11 +0,0 @@
-class AddOutcomes < ActiveRecord::Migration[5.0]
```

Comment:
Dont delete existing migrations. To change existing structure, add new migrations